DigitalLearningSolutions.Web/Views/Shared/SearchablePage/_Search.cshtml

```diff
@@ -18,7 +18,14 @@
       @Model.SearchLabel
     </label>
 
-        <input class="search-box nhsuk-input" id="search-field" name="searchString" type="search" placeholder="Search by activity name" autocomplete="off" value="@Model.SearchString" />
+        @if (Model.SearchLabel == "Search courses")

```

Comment:
I think we should use this Model.SearchLabel as the placeholder text - we will need to update it everywhere that this partial is used to ensure an appropriate SearchLabel is being passed. We could always set to "Search" everywhere but here, for now. What do you think?